nway/schemas.py

```diff
@@ -89,6 +89,7 @@ def hungarian_warn(self, data):
             logger.warning("Hungarian method not recommended. It is not "
                            "stable under permutations for typical cell "
                            "matching. Use Blossom.")
+        return data
```

Comment:
For stable marshmallow 3.0 the `post_load` function should also accept `**kwargs`.

See: https://marshmallow.readthedocs.io/en/stable/changelog.html#rc7-2019-06-15

This is fine for rc6, but if you want the ability to upgrade further it might not be a bad idea to just resolve now.

Reply:
Overall looks reasonable, I approve.

Reply:
added `**kwargs`